test/registry/jurors-registry-slashing.js

```diff
@@ -1,27 +1,185 @@
+              assert.equal(previousDeactivationBalance.toString(), currentDeactivationBalance.toString(), 'rewarded juror deactivation balance does not match')
+            })
+
+            it('slashes the active balances of the not rewarded jurors', async () => {
+              const [firstJurorPreviousActiveBalance, firstJurorPreviousAvailableBalance, firstJurorPreviousLockedBalance, firstJurorPreviousDeactivationBalance] = await registry.balanceOf(juror)
+              const [thirdJurorPreviousActiveBalance, thirdJurorPreviousAvailableBalance, thirdJurorPreviousLockedBalance, thirdJurorPreviousDeactivationBalance] = await registry.balanceOf(thirdJuror)
+
+              await registryOwner.slashOrUnlock(jurors, lockedAmounts, rewardedJurors)
+
+              const [firstJurorCurrentActiveBalance, firstJurorCurrentAvailableBalance, firstJurorCurrentLockedBalance, firstJurorCurrentDeactivationBalance] = await registry.balanceOf(juror)
+              assert.equal(firstJurorPreviousLockedBalance.minus(DRAFT_LOCK_AMOUNT.mul(3)).toString(), firstJurorCurrentLockedBalance.toString(), 'first slashed juror locked balance does not match')
+              assert.equal(firstJurorPreviousActiveBalance.minus(DRAFT_LOCK_AMOUNT.mul(3)).toString(), firstJurorCurrentActiveBalance.toString(), 'first slashed juror active balance does not match')
+              assert.equal(firstJurorPreviousAvailableBalance.toString(), firstJurorCurrentAvailableBalance.toString(), 'first slashed juror available balance does not match')
+              assert.equal(firstJurorPreviousDeactivationBalance.toString(), firstJurorCurrentDeactivationBalance.toString(), 'first slashed juror deactivation balance does not match')
+
+              const [thirdJurorCurrentActiveBalance, thirdJurorCurrentAvailableBalance, thirdJurorCurrentLockedBalance, thirdJurorCurrentDeactivationBalance] = await registry.balanceOf(thirdJuror)
+              assert.equal(thirdJurorPreviousLockedBalance.minus(DRAFT_LOCK_AMOUNT.mul(6)).toString(), thirdJurorCurrentLockedBalance.toString(), 'second slashed juror locked balance does not match')
+              assert.equal(thirdJurorPreviousActiveBalance.minus(DRAFT_LOCK_AMOUNT.mul(6)).toString(), thirdJurorCurrentActiveBalance.toString(), 'second slashed juror active balance does not match')
+              assert.equal(thirdJurorPreviousAvailableBalance.toString(), thirdJurorCurrentAvailableBalance.toString(), 'second slashed juror available balance does not match')
+              assert.equal(thirdJurorPreviousDeactivationBalance.toString(), thirdJurorCurrentDeactivationBalance.toString(), 'second slashed juror deactivation balance does not match')
+            })
+
+            it('does not affect the active balances of the current term', async () => {
+              let termId = await registryOwner.getLastEnsuredTermId();
```

Comment:
```suggestion
              let termId = await registryOwner.getLastEnsuredTermId()
```

we should consider adding a linter to tests!